modules/common-attributes.adoc

```diff
@@ -28,6 +28,7 @@ endif::[]
 :rh-storage-first: Red Hat OpenShift Container Storage
 :rh-storage: OpenShift Container Storage
 :rh-rhacm-first: Red Hat Advanced Cluster Management (RHACM)
+:rh-rhacs-first: Red Hat Advanced Cluster Security for Kubernetes (RHACS)
```

Comment:
**praise:** Good one mate 🤠

Reply:
Hi, I'm confused because elsewhere we use product-title-short and product-title. Should we use the new variables going forward?

Reply:
Yes, moving forward you can cause this new variable. For existing instances of `product-title-short` and `product-title`, you can keep it as it is. Hope this clarifies your concern.